ipalib/parameters.py

```diff
@@ -130,6 +130,13 @@
 from ipapython.dnsutil import DNSName
 
 
+MAX_UINT32 = (1 << 32) - 1
+# JavaScript Number.MAX_SAFE_INTEGER / Number.MIN_SAFE_INTEGER
+# JSON cannot safely encode values outside this range as regular number
```

Comment:
JSON itself does not specify range or precision of the number type.  So it is really JavaScript's implementation (and others that depend on IEEE float implementations) that cannot safely handle such values.

Reply:
Do you want him to clarify the comment?

Reply:
That would be nice.  I don't consider it blocker though.

Reply:
Ok I'm going to ack it since this constrains IPA/python to the lowest-common denominator.